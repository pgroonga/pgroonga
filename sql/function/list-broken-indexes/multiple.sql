CREATE TABLE memos (
  content text
);

CREATE TABLE tags (
  name text
);

CREATE INDEX pgrn_memos_index ON memos USING PGroonga (content);
CREATE INDEX pgrn_tags_index ON tags USING PGroonga (name);

SELECT * FROM pgroonga_list_broken_indexes();

DROP TABLE memos;
DROP TABLE tags;
