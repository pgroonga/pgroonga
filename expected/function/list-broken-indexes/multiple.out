CREATE TABLE memos (
  content text
);
CREATE TABLE tags (
  name text
);
CREATE INDEX pgrn_memos_index ON memos USING PGroonga (content);
CREATE INDEX pgrn_tags_index ON tags USING PGroonga (name);
SELECT * FROM pgroonga_list_broken_indexes();
 pgroonga_list_broken_indexes 
------------------------------
 pgrn_memos_index
 pgrn_tags_index
(2 rows)

DROP TABLE memos;
DROP TABLE tags;
